data/developerhub/applications.json

```
@@ -376,6 +376,19 @@
       "complexity": ["basic"],
       "pro": true,
       "cloudpods": false
+    },
+    {
+      "title": "Deploying a Database Migration Pipeline and creating CDC with DMS, Kinesis, and RDS MariaDB",
```

Is there a reason for the emphasis on CDC in the title?

---

None. Let me remove that :)

---

I don't know if it rolls of the tongue as easily, but might be more accurate! :smile: 
```suggestion
      "title": "Deploying a Database Migration Pipeline with DMS, creating Replication Tasks from RDS MariaDB to Kinesis",
```